src/modules/lorem/index.ts

```diff
@@ -332,7 +361,49 @@ export class LoremModule {
 
     const method = this.faker.helpers.arrayElement(methods);
 
-    return `${this[method]()}`;
+    if (typeof options === 'number') {
+      options = { length: options };
+    }
+
```

Comment:
You can use https://fakerjs.dev/api/helpers.html#rangetonumber to simplify the code below:

```suggestion
    if (options.length == null) {
      return this[method]();
    }

    const length = faker.helpers.rangeToNumber(options.length);

```